janusgraph-core/src/main/java/org/janusgraph/core/schema/JanusGraphDefaultSchemaMaker.java

```diff
@@ -31,30 +33,96 @@
 public class JanusGraphDefaultSchemaMaker implements DefaultSchemaMaker {
 
     public static final DefaultSchemaMaker INSTANCE = new JanusGraphDefaultSchemaMaker();
+    private static final Logger log = LoggerFactory.getLogger(JanusGraphDefaultSchemaMaker.class);
+
+    private boolean loggingEnabled;
 
     private JanusGraphDefaultSchemaMaker() {
     }
 
+    @Override
+    public void enableLogging(Boolean enabled) {
+        if (Boolean.TRUE.equals(enabled)) {
+            loggingEnabled = true;
+        }
+    }
+
     @Override
     public Cardinality defaultPropertyCardinality(String key) {
         return Cardinality.SINGLE;
     }
 
+    @Override
+    public EdgeLabel makeEdgeLabel(EdgeLabelMaker factory) {
+        logWarn("Edge Label with given name does not exist: {}", factory.getName());
```

Comment:
Wouldn't it make sense to add the information to this log message that the edge label will now be created?

Reply:
Make sense. I simply didn't change the log message introduced by https://github.com/JanusGraph/janusgraph/pull/1918 so that behavior wouldn't change. But anyway, it would be better to improve the log message as well.